### PR TITLE
UX: Add icons to all section links in Sidebar

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: app/assets/stylesheets/common/base/sidebar-section-link.scss [hunk 1]
```diff
@@ -1,10 +1,14 @@
+:root {
+  --d-sidebar-section-link-prefix-margin-right: 0.4rem;
+  --d-sidebar-section-link-prefix-width: 20px;
```
Comment (tgxworld) [inline app/assets/stylesheets/common/base/sidebar-section-link.scss:3]: prefixes have a fixed width and height of 20px

**Comment by tgxworld** [comment]
@awesomerobot I made some changes so that we can support the image and text prefix used by chat. Can you review the changes I made? Thank you!

**Review by martin-brennan — APPROVED** [review]
LGTM, and the screenshots look great too :+1:

**Comment by tgxworld** [comment]
Thank you @martin-brennan

**Pull request merged**